### PR TITLE
New Template for kit 24.05 Release

### PR DESCRIPTION
**Pull request by kelaja**
## Description
This PR introduces better guidance for KIT representatives to participate the release process.
TRG 8.03 is not present, so it has been removed from the Template

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [X] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [X] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/4_KIT_release_checks.md
Comment (RolaH1t) [inline .github/ISSUE_TEMPLATE/4_KIT_release_checks.md:9]: ...for your KIT

File: .github/ISSUE_TEMPLATE/4_KIT_release_checks.md
Comment (RolaH1t) [inline .github/ISSUE_TEMPLATE/4_KIT_release_checks.md:19]: **Version to be included in Eclipse Tractus-X release**: *version placeholder*

File: .github/ISSUE_TEMPLATE/4_KIT_release_checks.md
Comment (RolaH1t) [inline .github/ISSUE_TEMPLATE/4_KIT_release_checks.md:18]: insert line above: Best practice: have a professional Peer review your prepared content before you submit for expert approvals.

File: .github/ISSUE_TEMPLATE/4_KIT_release_checks.md
Comment (FaGru3n) [inline .github/ISSUE_TEMPLATE/4_KIT_release_checks.md:6]: Hi @kelaja,

could you please remove this item this will make the template not usable, because `milestone` is not a default value for issue templates

**Review by FaGru3n — CHANGES_REQUESTED** [review]
Hi @kelaja,

in general do we really need a dedicated release issue for Kit´s and is there an other process that the usual products?

and `milestone` as default for this template will will not work

**Review by SebastianBezold — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/4_KIT_release_checks.md
Comment (kelaja) [reply .github/ISSUE_TEMPLATE/4_KIT_release_checks.md]: @FaGru3n Thanks for reminding me. I applied your proposal, kindly approve.

File: .github/ISSUE_TEMPLATE/4_KIT_release_checks.md
Comment (danielmiehle) [inline .github/ISSUE_TEMPLATE/4_KIT_release_checks.md:18]: Suggestion: Have your prepared content reviewed by a professional peer group before submitting it for committer approval.

File: .github/ISSUE_TEMPLATE/4_KIT_release_checks.md
Comment (danielmiehle) [inline .github/ISSUE_TEMPLATE/4_KIT_release_checks.md:18]: KIT TRGs will be published next week. We can add the link later.

File: .github/ISSUE_TEMPLATE/4_KIT_release_checks.md
Comment (danielmiehle) [inline .github/ISSUE_TEMPLATE/4_KIT_release_checks.md:31]: KIT TRGs will be published next week. We can add the link later.

**Review by SujitMBRDI — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/4_KIT_release_checks.md
Comment (RolaH1t) [reply .github/ISSUE_TEMPLATE/4_KIT_release_checks.md:18]: text ok for me

File: .github/ISSUE_TEMPLATE/4_KIT_release_checks.md
Comment (RolaH1t) [reply .github/ISSUE_TEMPLATE/4_KIT_release_checks.md:18]: no additional link required in this section of the template

File: .github/ISSUE_TEMPLATE/4_KIT_release_checks.md
Comment (RolaH1t) [reply .github/ISSUE_TEMPLATE/4_KIT_release_checks.md:31]: IMHO the text and the existing link are ok and sufficient here - IF the KIT TRGs next week will end up in the same area (that is my assumption).
Proposal: delete the text in (brackets)
=> therefore the template is ready to go

**Review by RolaH1t — APPROVED** [review]

**Comment by RolaH1t** [comment]
...as KIT teams are expecting this template to be available cw14 => may I ask you guys for review and final confirmation @danielmiehle & @maximilianong 
afterwards @FaGru3n please merge

**Review by danielmiehle — COMMENTED** [review]
LGTM. KIT TRGs will most likey be 9.X and not 8.X! However, we have to insert the link as soon as the TRG ist ready anyway...

**Review by danielmiehle — APPROVED** [review]

**Review by SebastianBezold — APPROVED** [review]

**Review by FaGru3n — APPROVED** [review]
LGTM

**Pull request merged**
